smac_planner/src/node_se2.cpp

```diff
@@ -256,26 +266,27 @@ float NodeSE2::getHeuristicCost(
   to[1] = goal_coords.y;
   to[2] = goal_coords.theta * _motion_model.bin_size;
 
-  return _motion_model.state_space->distance(from(), to());
+  const float model_heuristic = _motion_model.state_space->distance(from(), to());
+  const float euclidean_heuristic = hypotf(
+    goal_coords.x - node_coords.x,
+    goal_coords.y - node_coords.y);
+  return NodeSE2::neutral_cost * std::max(euclidean_heuristic, model_heuristic);
```

Comment:
The Dubins or Reeds-Shepp distance can never be less than the Euclidean distance. They can be equal for a straight drive, and the model heuristic will get larger if turning is involved.

Reply:
I did this based on the paper saying that they take the maximum of the two. I would generally agree with you though, I had a sheet of paper I couldn't think of an example when it would be better. If you can't think of an example of that either, I can remove it.